### PR TITLE
Milestone - 2 , 3 and ADT templates

### PR DESCRIPTION
**Pull request by shashanknarang27**
We have added the milestone-2 , 3 and ADT templates

### CODE REVIEW

File: release.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ trigger:
 pr:
 - dotliquid
 - main
-- xoriant/*
+- templates/*
```
Comment (BoyaWu10) [inline release.yml:11]: It looks we already have this change in `microsoft:templates/m2` ([ref](https://github.com/microsoft/FHIR-Converter/blob/1268ed7913fbd210fa5b3dd336899f48f4de962b/release.yml#L11)). Are we using the latest code from `microsoft:templates/m2`?

Comment (shashanknarang27) [reply release.yml:11]: Yes , we are using the latest code from microsoft:templates/m2

File: src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs [hunk 2]
```diff
@@ -35,22 +35,64 @@ public static IEnumerable<object[]> GetDataForHl7v2()
...
+
+                new[] { "MDM_T01", "MDM-T01-01.hl7",  @"MDM-T01-01-expected.json"},
+                new[] { "MDM_T01", "MDM-T01-02.hl7",  @"MDM-T01-02-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-01.hl7",  @"MDM-T02-01-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-02.hl7",  @"MDM-T02-02-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-03.hl7",  @"MDM-T02-03-expected.json"},
+
+                new[] { "OML_O21", "OML_O21_01.hl7",  @"OML_O21-01-expected.json"},
+                new[] { "OML_O21", "OML_O21_02.hl7",  @"OML_O21-02-expected.json"},
+
```
Comment (BoyaWu10) [inline src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs:95]: Nit: Can we unify the file names of the sample data and expected results? Currently some file names are using `-` to combine words while some others are using `_`. Some of them are named `template_{number}-expected.json` while some others are named `template-expected-{number}`.

Comment (shashanknarang27) [reply src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs:95]: Sure , We will unify the file names and will push the changes for it

Comment (shashanknarang27) [reply src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs:95]: @BoyaWu10 : We have unified the name of the sample data and expected results and pushed the changes for it

Comment (BoyaWu10) [reply src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs:95]: Thanks! This looks much clearer.

**Review by BoyaWu10 — APPROVED** [review]

**Review by qiwjin — APPROVED** [review]

**Pull request merged**